slides/08-Platform-Hub/1060-HelpAndSupport.md

```diff
@@ -0,0 +1,6 @@
+## Where can I find some help
+
+ - Help and support have they're own section at the bottom of the platform hub task bar.
```

Comment:
~~they're~~their though its probably better to say: `Help and support is located in the section at the bottom of the platform hub task bar.`